### ISSUE TITLE
error: "Could not create client: directory missing new registration URL"

### ISSUE THREAD

**Issue by qazzyl** [xenolf/lego#562]
Hello,

I'm using AWS lightsail with instance *Bitnami WordPress*
I tried this tutorial: https://docs.bitnami.com/aws/how-to/generate-install-lets-encrypt-ssl/

This was code, I inputed
```
cd /tmp
curl -s https://api.github.com/repos/xenolf/lego/releases/latest | grep browser_download_url | grep linux_amd64 | cut -d '"' -f 4 | wget -i -
tar xf lego_linux_amd64.tar.xz
sudo mv lego /usr/local/bin/lego

sudo /opt/bitnami/ctlscript.sh stop
sudo lego --email="EMAIL-ADDRESS" --domains="DOMAIN" --path="/etc/lego" run
```

but this was a error
`Could not create client: directory missing new registration URL`

**Comment by johnschoofs** [comment]
+1 Having the same issue.

**Comment by mholt** [comment]
Probably just hitting scheduled maintenance: https://letsencrypt.status.io/

Feel free to re-open if this persists tomorrow!

**Comment by cpu** [comment]
> Probably just hitting scheduled maintenance: https://letsencrypt.status.io/

quick-note: the maintenance window won't start until 18:00 UTC. There's no currently known interruption in service and I suspect the root cause here is something other than maintenance.

**Comment by mholt** [comment]
Hm, you're right, I did my mental time zone conversion wrong.

What do the full logs show?

**Comment by johnschoofs** [comment]
How can I help to get you information you need @mholt? Not sure where log files are being written, is there somewhere that you could point me to?

**Comment by xenolf** [comment]
New binaries where this is resolved are now available with the v1.0.1 release. 
Thank you for reporting!
